pandas_gbq/gbq.py

```diff
@@ -2,11 +2,15 @@
 import logging
 import os
 import time
+import uuid
 import warnings
 from datetime import datetime
+from io import BytesIO
 from time import sleep
 
 import numpy as np
+import pandas as pd
+from google.cloud.bigquery.job import ExtractJobConfig
```

Comment:
This should go inside the method where it's used so that `_test_google_api_imports()` has a chance to run.